lib/op25_repeater/lib/p25_frame_assembler_impl.cc

```diff
@@ -30,7 +30,6 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
-#include <errno.h>
```

Comment:
Why are you removing this? OP25 has this included so it is easier to merge updates if I keep it in

Reply:
Line 28 is identical to that one. Is there supposed to be two duplicate ones?

Reply:
Ah! cool - it just didn't show up in the change review